# goal definition
float32 des_x
float32 des_y
---
# result definition
bool success
---
# feedback definition
float32 current_x
float32 current_y
